_posts/0000-01-02-jaca22-nica.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
This image represents how I feel doing this task



![Image of Yaktocat](https://octodex.github.com/images/yaktocat.png)

Reply:
- @githubteacher I’m lost here help me out please

Reply:
> This image represents how I feel doing this task
> 
> 
> 
> ![Image of Yaktocat](https://octodex.github.com/images/yaktocat.png)

Frustrating

Reply:
![Image of Yaktocat](https://octodex.github.com/images/yaktocat.png)
Not a present image